### PR TITLE
[app_recorder] Add rosbag converter plugins

### PR DESCRIPTION
**Pull request by 708yamaguchi**
Depends on https://github.com/jsk-ros-pkg/jsk_common/pull/1738
Tests will pass after the above PR is released.

I add plugin to convert rosbag into audio or video file.

With this script, we can generate video files from rosbag after the app.
I this case, we do not have to generate the video files while the app is running.

This can reduce CPU usage during app compared to audio_video_recorder.

### CODE REVIEW

**Comment by 708yamaguchi** [comment]
@iory 
Could you review this?

File: app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            str(plugin_args['rosbag_title']))
+        kwargs = {'wav_outpath': str(plugin_args['audio_path']),
+                  'topic_name': str(plugin_args['audio_topic_name']),
+                  'samplerate': int(plugin_args['audio_sample_rate']),
+                  'channels': int(plugin_args['audio_channels'])}
+        try:
+            bag_to_audio(rosbag_file_path, **kwargs)
+        except ValueError as e:
+            # topic is not included in bagfile
+            print('{}'.format(e))
```
Comment (iory) [inline app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py]: `rospy.logerr` is better?

File: app_recorder/src/app_recorder/rosbag_video_converter_plugin.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        if 'audio_topic_name' in plugin_args:
+            audio_kwargs = {'audio_topic': str(plugin_args['audio_topic_name']),
+                            'samplerate': int(plugin_args['audio_sample_rate']),
+                            'channels': int(plugin_args['audio_channels'])}
+            kwargs.update(audio_kwargs)
+        try:
+            bag_to_video(rosbag_file_path, **kwargs)
+        except ValueError as e:
+            # topic is not included in bagfile
+            print(e)
```
Comment (iory) [inline app_recorder/src/app_recorder/rosbag_video_converter_plugin.py]: `rospy.logerr` is better?

**Comment by knorth55** [comment]
Linter failed:

```
/github/workspace/app_recorder/src/app_recorder/rosbag_video_converter_plugin.py:21:80: E501 line too long (80 > 79 characters)
/github/workspace/app_recorder/src/app_recorder/rosbag_video_converter_plugin.py:22:80: E501 line too long (80 > 79 characters)
```

File: app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py (hunk 1 above)
Comment (708yamaguchi) [reply app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py]: Thank you! 
I updated in [`e691b49` (#54)](https://github.com/knorth55/app_manager_utils/pull/54/commits/e691b49d048cbf3589863313714aabb4093ca4a9)

File: app_recorder/src/app_recorder/rosbag_video_converter_plugin.py (hunk 2 above)
Comment (708yamaguchi) [reply app_recorder/src/app_recorder/rosbag_video_converter_plugin.py]: Thank you! 
I updated in [`e691b49` (#54)](https://github.com/knorth55/app_manager_utils/pull/54/commits/e691b49d048cbf3589863313714aabb4093ca4a9)

**Comment by 708yamaguchi** [comment]
@knorth55 

Thank you!
I fixed the linter errors in [`d2154e0` (#54)](https://github.com/knorth55/app_manager_utils/pull/54/commits/d2154e043c1aa3f6c785335e636b989662606df1)

File: app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+from jsk_rosbag_tools.bag_to_audio import bag_to_audio
```
Comment (knorth55) [inline app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py]: can you add try exception for this import?
if importerror, you can exit from this file and not importing this file.

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py (hunk 3 above)
Comment (knorth55) [reply app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py]: In that case, we dont need to update .ci.rosinstall

Comment (708yamaguchi) [reply app_recorder/src/app_recorder/rosbag_audio_converter_plugin.py]: Thank you for your advice.

I reverted ci.rosinstall update in [`634c1d3` (#54)](https://github.com/knorth55/app_manager_utils/pull/54/commits/634c1d38dc2c1b88ca8c2a6a163c23fc8bb6652d)

I added try-except and exit code in [`3a0f786` (#54)](https://github.com/knorth55/app_manager_utils/pull/54/commits/3a0f786cf40c0b750035d9513a79c2bc1d968036)

**Pull request merged**